pulse/inngest-router/README.md

```diff
@@ -0,0 +1,125 @@
+# Prisma Pulse-Inngest Router
+
+This package enables event-driven support for your existing Inngest workflows, using Prisma Pulse. Trigger workflows instantly whenever a database table changes, so you can create timely workflows that run instantly -- not only batch workflows.
+
+## Pre-requisites
+
+- An existing project with [Inngest](https://www.inngest.com/docs) and [Prisma Client](https://www.prisma.io/docs/getting-started) configured
+- A **Pulse API key** which you can get by enabling Pulse in a project in your
+  [Prisma Data Platform](https://pris.ly/pdp) account (learn more in the
+  [docs](https://www.prisma.io/docs/platform/concepts/environments#api-keys))
+- An existing [Inngest](https://www.inngest.com) account and
+  [Event Key](https://www.inngest.com/docs/events/creating-an-event-key?ref=environment-variables)
+
+## Get started
+
+### Fork or clone this repo
+
+```bash
+npx try-prisma@latest --template pulse/inngest-router
+```
+
+You should operate with your own version of this service in a GitHub repository that you own. This allows you to track changes, and deploy to cloud platforms easily.
+
+### Set up your API keys
+
+Make a copy of the `.env.example` file, rename it to `.env`, and update it with your own keys and database connection string:
+
+```bash
+cp .env.example .env
+```
+
+### Add your Prisma schema
+
+Add your existing Prisma schema file to the `/prisma` directory, overwriting the existing example schema.
+
+If you don't already have a project using Inngest and Prisma Client, you may choose to try another [Prisma example](https://github.com/prisma/prisma-examples) instead.
```

Comment:
Up to you, but just to understand: 

Should we maybe rephrase this?
```suggestion
If you don't already have a project using Inngest and Prisma Client, you can choose another [Prisma example](https://github.com/prisma/prisma-examples) to start with instead.
```

Reply:
Actually, on second thought. Maybe we phrase this a different way since this repo has a sample `prisma.schema`
```suggestion
If you don't already have a project using Inngest and Prisma Client, you can use the sample Prisma Schema provided in this repo. Run `npx prisma db push` to push this schema to a test database of your choosing. [Check out the db push docs for more info](https://www.prisma.io/docs/orm/reference/prisma-cli-reference#db-push).

You can also try one of our other [Prisma examples](https://github.com/prisma/prisma-examples) as a starting point instead.
```